build_image.sh

```diff
@@ -170,6 +184,13 @@ elif [ "$IMAGE_TYPE" = "aboot" ]; then
     zip -g $ABOOT_BOOT_IMAGE version
     rm version
 
+    generate_device_list ".platforms_asic"
+    # Check file existence in case of general build
```

Comment:
>Check file existence in case of general build [](http://example.com/codeflow?start=6&length=45)

What do you mean by "general build"? Is this file always existing after previous generating function?

Reply:
I think the file doesn't exist in some case. For example, if the TARGET_MACHINE was set to 'general', it won't match any platform_asic. I will bypass the check here.